src/shared/lib/timers/hooks/useDelayedInterval.ts

```diff
@@ -1,24 +1,33 @@
-import { useCallback, useRef } from 'react';
+import { useCallback, useEffect, useRef } from 'react';
 
-import { useInterval, useOnForegroundDebounced, useOnceRef } from '../../';
+import {
+  Logger,
+  useInterval,
+  useOnForegroundDebounced,
+  useOnceRef,
+} from '../../';
+import useOnBackground from '../../hooks/useOnBackground';
 
-const Interval = 30000;
+const Interval = 60000;
 
 const Delay = 5000;
 
-type Props = {
-  onTick: () => unknown;
-};
-
-const useDelayedInterval = ({ onTick }: Props) => {
+const useDelayedInterval = (onTick: () => void) => {
   const onTickRef = useRef(onTick);
   onTickRef.current = onTick;
 
   const onIntervalPass = useCallback(() => {
     onTickRef.current();
   }, []);
 
-  const { start, stop } = useInterval(onIntervalPass, Interval);
+  const { start, stop } = useInterval(
+    () => {
+      Logger.log('[useDelayedInterval:useInterval] Tick');
```

Comment:
I am pretty sure it will cause an influx of these lines in the logs

Reply:
I don't think so, once per minute when app in foreground